src/debug/HyprCtl.cpp

```diff
@@ -14,6 +14,32 @@
 #include <sstream>
 #include <string>
 
+std::string activeWorkspaceRequest(HyprCtl::eHyprCtlOutputFormat format) {
```

Comment:
if we're gonna do this then look at window requests and unify the data getting

Reply:
Alright, I'll make it like window requests when I get home

Reply:
resolved in [`a7c6d92` (#2202)](https://github.com/hyprwm/Hyprland/pull/2202/commits/a7c6d928613d941933d0f3a5ab7fe00417633516)